lib/metababel/bt2_trace_class_generator.rb

```diff
@@ -255,7 +255,7 @@ def initialize(parent:, name: nil, specific_context_field_class: nil, payload_fi
 
       @parent = parent
       @name = name
-      raise 'Name is mandatory for BTEventClass' if name.nil? and !rec_trace_class.match
+      raise 'Name is mandatory for BTEventClass' if name.nil? && !rec_trace_class.match
```

Comment:
Same here, the change makes no difference

Reply:
Yep, we will always use `&&` and `||` so we don't need to think. I think it's just a best practice to be consistent.